assets/in

```diff
@@ -73,8 +73,10 @@ cd $destination
 
 git fetch origin refs/notes/*:refs/notes/*
 
-concourse_git_resource__depth="$depth" \
-  "$bin_dir"/deepen_shallow_clone_until_ref_is_found "$ref"
+if [ "$depth" -gt 0 ]; then
+  concourse_git_resource__depth="$depth" \
+    "$bin_dir"/deepen_shallow_clone_until_ref_is_found "$ref"
+fi
 
 git checkout -q $ref
```

Comment:
This belongs in an `else` for the above `if` - it's not an error, exactly, but it's superfluous.

Reply:
Fixed in ad553ecb6aa971840fb6be0b805a50616d2b2480.